test/specs/from.coffee

```diff
@@ -0,0 +1,30 @@
+Observable = require('zen-observable')
+Rx = require('@reactivex/rxjs')
```

Comment:
It's a [scoped package](https://docs.npmjs.com/misc/scope) – a relatively new feature in NPM that seems to be only available for 0.12 and newer nodes.